app/renderer/components/common/modalOverlay.js

```diff
@@ -71,6 +72,18 @@ class ModalOverlay extends ImmutableComponent {
           [customTitleClassesStr]: true
         })} data-l10n-id={this.props.title}
         /> : null)
+      subTitle = (this.props.subTitle
+        ? <aside className={css(styles.dialog__header__subTitle)}>
+          <div className={cx({
+            [css(styles.dialog__header__subTitle_text)]: true,
+            [customTitleClassesStr]: true
```

Comment:
(TODO: we would remove `customTitleClassesStr` etc. later to avoid `cx` which sometimes prevents properties from cascading properly)